commercial/app/model/feeds/FeedReader.scala

```diff
@@ -33,7 +33,6 @@ class FeedReader(wsClient: WSClient) extends Logging {
       val requestHolder: WSRequest = {
         val unsignedRequestHolder: WSRequest = wsClient.url(request.url)
           .withQueryString(request.parameters.toSeq: _*)
-          .withRequestTimeout(request.timeout)
```

Comment:
@JonNorman Perhaps do away with direct book lookups altogether?  Instead, keep a persistent cache of book data in a dynamo table and find the books in there.  Traverse the table and update stale data as a background process.